GirafRest/Controllers/AccountController.cs

```diff
@@ -167,11 +167,18 @@ public async Task<ActionResult> Register([FromBody] RegisterDTO model)
             {
                 if (department != null)
                 {
-                    if (model.Role == GirafRoles.Citizen)
-                        AddGuardiansToCitizens(user);
-                    else if (model.Role == GirafRoles.Guardian)
-                        AddCitizensToGuardian(user);
-                    // save changes
+                    switch (model.Role)
+                    {
+                        case GirafRoles.Citizen:
+                            AddGuardiansToUser(user);
```

Comment:
should this not be addCitizenToGuardian?

Reply:
and why is it not addCitizenToUser (when the others are "ToUser")

Reply:
Har selv undret mig meget over koden. Vi har ikke ændret den orginale funktionalitet, da det ikke er i scope af https://github.com/aau-giraf/weekplanner/issues/625.

Har ændret `AddGuardianToCitzen` til `AddGuardianToUser` for at kunne benytte samme metode når man opretter en trustee. Som det er nu i giraf er der nemlig kun logik for en 'guardian & citizen relation' (`GuardianRelation.cs`). Vi mener at dette skal løses i forbindelse med et andet issue.

> should this not be addCitizenToGuardian?

Det der sker her er at; når man registrer en citizen bliver alle guardians indenfor samme department, tilføjet til denne citizen. Tror at `addCitizenToGuardian` vil være lidt misvisende grundet dette.

> and why is it not addCitizenToUser (when the others are "ToUser")

Tror også dette er misvisende grundet overstående. 
Måske vil det være mere tydeligt at navngive metoden `addCitizensToUser`, hvad synes i om det?

Reply:
synes helt klart at den addCitizensToGuardian skal hedde addCitizensToUser

Reply:
Done